spec/lib/web_of_science/harvester_spec.rb

```diff
@@ -13,7 +13,7 @@
 
   after(:all)  { savon.unmock! }
 
-  let(:author) { create :russ_altman }
+  let(:author) { create :russ_altman, :blank_orcid } # use a blank orcid, because these mocks work with the name query
   let(:wos_uids) { %w[WOS:A1976BW18000001 WOS:A1972N549400003] } # from wos_retrieve_by_id_response.xml
```

Comment:
this spec does not use VCR and is mocked to run only name queries, so will not behave correctly if we let it run ORCID queries as well -- since it is basically supposed to test the logic of the harvester class itself, it seems sufficient to verify it continues to make the call to our `QueryAuthor` class, which actually runs the underlying name or orcid queries as needed.  These class all have updated tests below that use VCR to test all of the new logic implemented here...